### PR TITLE
4367 bug fixes

### PR DESCRIPTION
**Pull request by matthopson**

### CODE REVIEW

File: shared/src/business/entities/Practitioner.js [hunk 1]
```diff
@@ -58,31 +58,67 @@ const VALIDATION_ERROR_MESSAGES = {
...
+    .allow(null)
+    .description('An alternate phone number for the practitioner.'),
+  admissionsDate: joi
+    .date()
+    .iso()
+    .max('now')
+    .required()
+    .description(
+      'The date the practitioner was admitted to the bar in their state.',
+    ), // TODO: Verify what this actually is
```
Comment (matthopson) [inline shared/src/business/entities/Practitioner.js]: Changing this to, "The date the practitioner was admitted to the Tax Court bar." per convo in standup.

File: shared/src/business/useCases/users/createPractitionerUserInteractor.js [hunk 2]
```diff
@@ -29,10 +29,16 @@ exports.createPractitionerUserInteractor = async ({
       initials: user.lastName.charAt(0) + user.firstName.charAt(0),
     },
   );
+
+  // The NewPractitioner entity splits the name. Combine them to
+  // match our User / Practitioner convention of a single field
+  const name = `${user.firstName} ${user.lastName}`;
```
Comment (sutt0n) [inline shared/src/business/useCases/users/createPractitionerUserInteractor.js]: Should this be an entity function? e.g. `getFullName()` or similar?

File: web-client/src/presenter/actions/createPractitionerUserAction.js [hunk 3]
```diff
@@ -28,7 +28,7 @@ export const createPractitionerUserAction = async ({
     return path.success({
       alertSuccess: {
         message:
-          'The user has been notified via the email addressed provided that they can no login to the system. You may continue creating Practitioner Users with the form below.',
+          'The user has been notified via the email addressed provided that they can now login to the system. You may continue creating Practitioner Users with the form below.',
```
Comment (sutt0n) [inline web-client/src/presenter/actions/createPractitionerUserAction.js]: `addressed` -> `address`

**Review by sutt0n — CHANGES_REQUESTED** [review]
Requesting changes for a possible typo (if all good, then I'll approve), but had a question as well.

**Review by sutt0n — APPROVED** [review]

**Review by rachaelparris — APPROVED** [review]

**Pull request merged**